src/sentry/remote_config/docs/protocol.md

```diff
@@ -79,16 +144,73 @@ If the server's ETag does not match the request's a 200 response is returned.
       "features": [
         {
           "key": "hello",
-          "value": "world"
+          "value": "world",
+          "variants": [
+            {
+              "rollout": {
+                "percentage": 50,
+                "sticky": {
+                  "group": "5f927c1c3676abbe5f13d9f8d28ffa625e80bf04",
+                  "target": "user_id"
+                }
+              },
+              "rules": [
+                {
+                  "operator": "==",
+                  "property": "region",
+                  "value": "Europe"
+                },
+                {
+                  "operator": ">=",
+                  "property": "now",
+                  "value": 1717684416.495
+                }
+              ],
+              "value": "computer"
+            }
+          ]
         },
         {
           "key": "has_access",
-          "value": true
+          "value": true,
+          "variants": []
         }
       ],
       "options": {
-        "sample_rate": 1.0,
-        "traces_sample_rate": 0.5
+        "sample_rate": {
+          "value": 0.1,
+          "variants": [
+            {
+              "rollout": null,
+              "rules": [
+                {
+                  "operator": "==",
+                  "property": "device_family",
```

Comment:
Based on my suggestion above, here's an example where we'd get the value off of the scope, and not (necessarily unless overwritten?) from the user context parameter to the `getConfig` call:

```suggestion
                  "property": "{{sentry.device_family}}",
```